pages/news/[slug].js

```diff
@@ -0,0 +1,127 @@
+import Head from 'next/head'
+import styles from '../../styles/News.module.css';
+import { useRouter } from 'next/router';
+import { Header } from '../../components/Header'
+import { Footer } from '../../components/Footer'
+
+
+// • News code was adapted from the Next.js news site tutorial: https://youtu.be/xtItzwYG6oQ
+
+// • "Slug" is the unique identifying part of a web address, usually the endpoint: https://developer.mozilla.org/en-US/docs/Glossary/Slug
+
+// • The YouTube instructor for the news tutorial uses "window.open" (Line 34) for the onClick action, and the target "blank" has to be used as one of two arguments when a window location is used: https://stackoverflow.com/questions/18476373/how-to-add-target-blank-to-javascript-window-location
+
+// • Testing to see if news images should/should not go to edges of cards. For now, I used object-fit css class for images, and line-clamping css properties to only show the first 3 lines of article content.
+
+// • For now, I repeated "{styles.container}" for lines 26 and 30 in so that the dashed outline page title is consistent with other page titles.
+
+// • Since the news grid is not a component like the grids in the support and wins pages, the div hierarchy is slightly different to render consistent styling.
+
+
+export const News = ({ pageNumber, articles }) => {
+
+    const router = useRouter();
+
+    return (
+        <div className={styles.container}>
+        <Head><title>Start Asian Love</title></Head>    
+        <Header/>
+
+            <div className={styles.container}>
+                <h2>The Latest News</h2>
+            </div>
+            <div className='page-container'>
+                <div className={styles.main}>
+                    {articles.map((article, index) => (
+                        <div onClick={() => window.open(article.url,'_blank')} key={index} className={styles.post}>
```

Comment:
I'm curious, what does `window.open(article.url, 'blank')` do? Is there a difference between a regular `<a>` with an href vs using an `onClick` for this?

Reply:
Hm, I'm not sure of the difference either, and to be honest, the tutorial instructor didn't explain it, but I think it may have something to do with validation? Here's an article about the W3C HTML validation: https://osric.com/chris/javascript/javascript-anchor-target-blank.html

- Personally, I prefer links opening in a new browser tab, but this probably doesn't fall under best practices, and we can definitely omit '_blank' before launching.